docs/readmes/lte/deploy_install_docker.md

```diff
@@ -0,0 +1,129 @@
+---
+id: deploy_install_docker
+title: Install Docker AGW
+hide_title: true
+---
+
+# Install Docker based Access Gateway on Ubuntu (Bare Metal)
+
+## Prerequisites
+
+To set up a Magma Access Gateway, you will need a machine that
+satisfies the following requirements:
+
+- AGW_HOST: 64bit-X86 machine, baremetal strongly recommended
```

Comment:
What is the reason behind this strong recommendation? Note that in `agw_install_docker.sh` it says "Reboot this VM to apply kernel settings", so we should probably replace "VM" by "machine" or "system" there.

Reply:
You are right, let me update this.